app/src/main/java/memphis/myapplication/UI/LoginFragment.java

```diff
@@ -162,6 +162,7 @@ private int loginAttempt(String domain, String username, String password) {
         protected Void doInBackground(Void... voids) {
             // save username and go to mainpage
             SharedPrefsManager.getInstance(getActivity()).setCredentials(username, password, domain);
+            Globals.setUsername(username);
             return null;
```

Comment:
Is return null needed here?

Reply:
This is from one of the GSoC applicants, but from I believe so. doInBackground has to return something. The method's return type is Void, so it may be possible to return Void. Either way, it's never touched again.